requirements-min.txt

```
@@ -2,8 +2,9 @@
 h5py==2.10  # support for selection of datasets with list of indices added in 2.10
 importlib-metadata==4.2.0; python_version < "3.8"  # TODO: remove when minimum python version is 3.8
 importlib-resources==5.12.0; python_version < "3.9"  # TODO: remove when when minimum python version is 3.9
-jsonschema==2.6.0
+jsonschema>=3.2.0
```

```suggestion
jsonschema==3.2.0
```
These minimum requirements should all be pinned to exact values. If a higher version is allowed, it will be used, and we will no longer be testing whether hdmf works with these minimum requirements.